### PR TITLE
Sparkle Fix Textarea auto scroll when text is very long

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Eng runner card: https://github.com/dust-tt/tasks/issues/593

This fixes an auto-scroll weird behavior when text is very long on a text area.

### CODE REVIEW

**Comment by PopDaph** [comment]
Wait it's not perfect when I cut text from a big Textarea

**Comment by fontanierh** [comment]
@PopDaph do you have any screenshots / GIFs of what the issue is and how it is after this fix ? I'm not quite sure what I am looking at here !

**Comment by fontanierh** [comment]
To be clear, it looks like some classes have been re-ordered but I would not expect any functional change from this ?

**Comment by PopDaph** [comment]
@fontanierh there is a video on the link provided in OP showing the bug. 

I will provide a video of the behavior after I added a commit to fix the height when we cut text so that you know what you're looking at 🙏🏻

**Comment by PopDaph** [comment]
Shared on slack!

File: sparkle/src/components/TextArea.tsx [hunk 1]
```diff
@@ -33,7 +32,7 @@ export function TextArea({
       <textarea
         ref={textareaRef}
         className={classNames(
-          "s-min-h-60 overflow-y-auto s-block s-w-full s-min-w-0 s-rounded-xl s-text-sm s-transition-all s-duration-200 s-scrollbar-hide",
```
Comment (PopDaph) [inline sparkle/src/components/TextArea.tsx:36]: No change here it's my linter who reordered the classes!

**Review by fontanierh — APPROVED** [review]
lgtm

**Pull request merged**